package.json

```
@@ -123,7 +124,7 @@
 		"yorkie": "^2.0.0"
 	},
 	"resolutions": {
-		"webpack-dev-middleware": "3.6.0"
+		"webpack-dev-middleware": "^3.6.1"
```

`yarn dev:docs` is crashing after this change.
```suggestion
		"webpack-dev-middleware": "3.6.0"
```
Vuepress Issue: https://github.com/vuejs/vuepress/issues/1417